### ISSUE TITLE
Level-3 Accessibility button

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#429]
This ticket is a part of #239 and has been priorritized

When the user goes to level-3 from a publication.

The focus should go to
1. Close button
2. Share button
3. Content on the web-page
4. Back and forward button at the bottom.

Note: The buttons at the bottom toolbar are missing their respective labels.
1. Back arrow at the bottom should labelled as "back" button
2. Forward arrow at the bottom should be labelled as "Forward" button

Important: When the focus goes to the bottom, it should identify "Toolbar" > "Back button" > "Forward button". If the button is disabled it should say "dimmed".
